python/lsst/sims/utils/treeUtils.py

```diff
@@ -0,0 +1,72 @@
+"""
+This file contains coordinate transformation methods and utilities for converting an ra,dec coordinate set to
+cartesian coordinates and to grid id using a spatial tree.
+"""
+from __future__ import division
+
+import numpy as np
+from scipy.spatial import cKDTree as kdtree
+
+__all__ = ['_treexyz', '_rad_length', '_buildTree']
+
+
+def _treexyz(ra, dec):
```

Comment:
There is a method `cartesianFromSpherical` here

https://github.com/lsst/sims_utils/blob/master/python/lsst/sims/utils/CoordinateTransformations.py#L167

Can you use that?